src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -283,7 +283,7 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, diagnost
       !
       ! start by adding the mininum background values to the viscocity/diffusivity arrays
       !
-      if (cvmixBackgroundOn) then
+      if (config_cvmix_background_scheme == 'constant') then
```

Comment:
@philipwjones and @mattdturner as we write new code, should we avoid string comparisons when possible? They are easier to read, but we can use logicals if it is better for performance.

Reply:
Per @philipwjones in this comment (https://github.com/MPAS-Dev/MPAS-Model/issues/394#issuecomment-553016388):

> Either way, please convert strings to other forms. Simple binary options should be implemented as logical flags. For conditions where we have multiple options, the preferred way would be to use the Fortran equivalent of enums - basically named integer parameters.

Reply:
Thanks @mattdturner, that is helpful. Can you point me to an example of enums? Is that when we use caps variables at the top of the module?

Reply:
@mark-petersen see for example how we treat equation of state choices

Reply:
I think the idea is to use strings in the namelist, then in `init` convert to integers and create new variables for the different values.  For example, keep `config_cvmix_background_scheme` as it is in the namelist.  Then in init
```
! Define the named integer parameters
integer :: config_cvmix_background_scheme_type
integer, parameter :: &
    backgroundSchemeNone = 0, &
    backgroundSchemeConstant = 1, &
    backgroundSchemeBryanLewis = 1

if config_cvmix_background_scheme == 'constant':
    config_cvmix_background_scheme_type = backgroundSchemeConstant
else if config_cvmix_background_scheme == 'bryanLewis':
    config_cvmix_background_scheme_type = backgroundSchemeBryanLewis
else if config_cvmix_background_scheme == 'none':
    config_cvmix_background_scheme_type == backgroundSchemeNone
```

Then use a `select case (convig_cvmix_background_scheme_type)` in the code

Reply:
Or reference equation of state like @philipwjones said (I wasn't aware of where else in the model this was done)

Reply:
I replaced these string comparisons w/ named integer parameters